includes/weather-functions.php

```diff
@@ -37,9 +37,21 @@ function get_weather( $cache_key ) {
 		}
 
 		if ( $json_url ) {
-			$json = array();
-			$args = array(
-				'timeout' => get_option( 'weather_service_timeout' )
+			// Setup curl request and execute. Log errors if necessary.
+
+			$dt = urlencode( microtime() );
+
+			if ( strpos( $json_url, '?' ) > -1 ) {
```

Comment:
`strpos` can return `false` if the `?` isn't present in the URL--I'm not sure off the top of my head if/how PHP will complain about performing a greater-than comparison against `false`, but it may make more sense to perform a strict `strpos( $json_url, '?' ) !== false` check here anyway.

Reply:
Updated.